### DISCUSSION TITLE
v1.3.0

### DISCUSSION THREAD

**Discussion by CloneWith** [CloneWith/opsu#107] [Announcements]
### Security

- Use TLS instead of SSL for network connection

### UI

- Fix bottom bar position issue in some special cases
- Remake & add settings and download buttons in the main menu
- Implement basic I18N framework (using Gettext PO)
- Use map to load needed font glyphs (especially for CJK fonts)
- Write skin changes to configuration file before reloading (to avoid data loss)

### Gameplay

- Use customized colour for experimental slider borders
- Allow for extraordinary difficulty settings (**staging**, feel free to discuss about this)

<hr /><em>This discussion was created from the release <a href='https://github.com/CloneWith/opsu/releases/tag/v1.3.0'>v1.3.0</a>.</em>

**Comment by JKP5758** [comment]
Emm, How to Run this game?

**Reply by CloneWith** [reply]
Hi, which platform are you currently using? Now opsu! supports Windows and Linux (tested).

For a general procedure of running this game you can refer to [this wiki page](https://github.com/CloneWith/opsu/wiki/Getting-Started).
